cli/loader/Cargo.toml

```diff
@@ -15,6 +15,7 @@ rust-version.workspace = true
 anyhow = "1.0"
 cc = "^1.0.58"
 dirs = "3.0"
+fs2 = "0.4.3"
```

Comment:
Why did you choice the `fs2` crate?

Reply:
brings in a cross-platform trait to lock files easily

Reply:
I meant it looks unmaintained and there are fs3, fs4 crates.